config/src/keyassignment.rs

```diff
@@ -457,6 +457,9 @@ pub struct QuickSelectArguments {
 #[derive(Debug, Clone, PartialEq, FromDynamic, ToDynamic)]
 pub struct PromptInputLine {
     pub action: Box<KeyAssignment>,
+    /// Optional label to pre-fill the input line with
+    #[dynamic(default)]
+    pub with_content: String,
```

Comment:
I'd prefer a name like this, and I think we should make this optional to work better with another suggestion I'm going to make

```suggestion
    pub initial_value: Option<String>,
```

Reply:
Moved all changes to a single commit. Thank you!